index.html

```diff
@@ -30,23 +30,23 @@ <h3 class="let">LET'S CONNECT</h3>
                 <ul class="nav-links">
                     <li class="lift"><a href="https://twitter.com/Ezechukwuchibu2"><img src="image-source/twitter.svg" alt="twitter"></a></li>
                     <li class="lift"><a href="https://www.linkedin.com/in/ezechukwu-chibuike/"><img src="image-source/Linkedin icon.svg" alt="Linkedin"></a></li>
-                    <li class="lift"><a href="#"><img src="image-source/medium.svg" alt="Vector-1"></a></li>
+                    <li class="lift"><a href="#"><img src="image-source/medium.svg" alt="Medium"></a></li>
                     <li class="lift"><a href="https://github.com/Moses-chibuike"><img src="image-source/github.svg" alt="github"></a></li>
-                    <li class="lift"><a href="#"><img src="image-source/angellist.svg" alt="Vector"></a></li>
+                    <li class="lift"><a href="#"><img src="image-source/angellist.svg" alt="angellist"></a></li>
                 </ul>
 
         </section>
         <section id="portfolio-works" class="Works">
             <div class="card port-one">
-                <!-- <div class="profile"><img src="image-source/Snapshoot Portfolio.png" alt="Snapshoot"></div> -->
-                <div class="profile"><img src="image-source/Snapshoot Portfolio.png" alt="Snapshoot"></div>
+                <!-- <div class="profile"><img src="image-source/Snapshoot Portfolio.png" alt="project 1"></div> -->
+                <div class="profile"><img src="image-source/Snapshoot Portfolio.png" alt="project 1"></div>
                 <div class="card-port-description">
                     <h3>Tonic</h3>
                     <div class="profile-description">
                         <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
+                        <img src="image-source/Counter.png" alt="">
                         <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
+                        <img src="image-source/Counter.png" alt="">
                         <p>2015</p>
```

Comment:
- [ ] Great job on adding the alt attribute here 🎉 it really makes your page more accessible. In some cases, it's ok to leave it empty and you did that well in some others but in this case, it's necessary to tell the reader that this is a  logo of ..., in case it's not rendered. 🤓